packages/valory/skills/liquidity_trader_abci/payloads.py

```diff
@@ -71,6 +71,20 @@ class DecisionMakingPayload(BaseTxPayload):
     content: str
 
 
+@dataclass(frozen=True)
+class DecideAgentStartingPayload(BaseTxPayload):
+    """Represent a transaction payload for the DecideAgentRound."""
+
+    vote: bool
+
+
+@dataclass(frozen=True)
+class DecideAgentEndingPayload(BaseTxPayload):
+    """Represent a transaction payload for the DecideAgentRound."""
+
+    vote: bool
```

Comment:
These are identical, should not be duplicated.

Reply:
combined into single payload